### ISSUE TITLE
Scroll-to-zoom Broken MS Edge

### ISSUE THREAD

**Issue by jakcharvat** [openscope/openscope#978]
Browser: MS Edge

Airport: Any

Approximate play time before issue: Any

#### Issue Description
Scroll-to-zoom doesn't work with a trackpad. I can scroll but the scope doesn't zoom

#### Steps to reproduce
- launch Edge
- browse to openscope
- Try to zoom in or out

**Comment by sam-irl** [comment]
Are you sure this is an Edge issue? Scrolling with a mouse works fine for me, and I know you specified a trackpad, but I'm curious as to whether or not this is with Edge or your trackpad. Do you have the same issue on another browser (such as Chrome?)

**Comment by erikquinn** [comment]
I have the issue on a touchpad with ms edge. haven't tried a mouse.

**Comment by jakcharvat** [comment]
Yes, It is an Edge issue. I generally test and play openScope in Chrome and works fine. Haven't tested Edge with a mouse yet
